distro/exporter.go

```diff
@@ -21,7 +21,7 @@ import (
 	"os"
 
 	"github.com/go-logr/logr"
-	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/exporters/jaeger" //nolint:staticcheck // Jaeger is deprecated, but we still support it to not break existing users.
```

Comment:
Can you open an issue to track our deprecation of this? IIRC it was deprecated in the GDI spec.

Reply:
I will create a deprecation log message as part of resolving https://github.com/signalfx/splunk-otel-go/issues/2484